packages/shared/src/components/cards/common/ClickbaitShield.tsx

```diff
@@ -87,8 +101,8 @@ export const ClickbaitShield = (): ReactElement => {
           )
         }
         iconSecondaryOnHover
-        onClick={() => {
-          // TODO: fetch smart title
+        onClick={async () => {
+          await fetchSmartTitle();
         }}
```

Comment:
Doesn't this work already?
```suggestion
        onClick={fetchSmartTitle}
```

Reply:
Yeah, works. Updated